improver/calibration/reliability_calibration.py

```diff
@@ -488,6 +488,178 @@ def process(self, cubes, coordinates=None):
         return result
 
 
+class ManipulateReliabiltiyTable(PostProcessingPlugin):
+    """
+    A plugin to manipulate the reliability tables to before they are used to
+    calibrate a forecast.
+    x and y must be collapsed
+    The result is a reliability diagram with monotonic observation frequency.
+    Steps taken are:
+        1.
+    """
+
+    def __repr__(self):
+        """Represent the configured plugin instance as a string."""
+        return "<ManipulateReliabiltiyTable>"
+
+    @staticmethod
+    def _sum_pairs(array, lower, upper):
+        """
+        Replace a pair of values in an array with their sum.
+
+        Args:
+            array (numpy.ndarray):
+                Array to be modified.
+            lower (int):
+                Lower index of pair.
+            upper (int):
+                Upper index of pair.
+
+        Returns:
+            numpy.ndarray:
+                Array where a pair of values has been replaced by their sum.
+        """
+        array[lower] = np.sum(array[lower : upper + 1])
+        return np.delete(array, upper)
+
+    def _create_new_bin_coord(self, reliability_table, lower, upper):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=6001543)